tool/tsh/tsh.go

```diff
@@ -273,6 +273,8 @@ const (
 
 	clusterHelp = "Specify the Teleport cluster to connect"
 	browserHelp = "Set to 'none' to suppress browser opening on login"
+
+	proxyDefaultResolutionTimeout = 2 * time.Second
```

Comment:
Is 2 seconds appropriate or recommended by an RFC, how was this value selected?

Reply:
It's from the RFC. Have added commentary to that effect.

Reply:
👍